utils/update-sveltekit.js

```diff
@@ -0,0 +1,112 @@
+const { execFileSync } = require('child_process');
+let { create } = require('create-svelte');
+const fs = require('fs');
+
+function exec(cmd, args, opts) {
+  console.log({ input: `${cmd} ${args.join(' ')}` });
+  const output = execFileSync(cmd, args, opts).toString().trim();
+  console.log({ output });
+  console.log();
+  return output;
+}
+
+function newVersion(packageName) {
+  return exec('npm', ['view', packageName, 'dist-tags.latest']);
+}
+
+const PACKAGE_JSON_PATH = '../examples/sveltekit/package.json';
+const SVELTEKIT_CONFIG_PATH = '../examples/sveltekit/svelte.config.js';
+
+/**
+ * Update the SvelteKit example. We care about three packages: SvelteKit,
+ * `adapter-vercel`, and `create-svelte`. If SvelteKit or `adapter-vercel` have changed,
+ * we can simply run `create-svelte` to update the example. If `create-svelte` has
+ * changed, we need to upgrade it, then run it to update the example.
+ */
+module.exports = async ({ github, context }) => {
+  const { '@sveltejs/kit': oldKit, '@sveltejs/adapter-vercel': oldVercel } =
+    require(PACKAGE_JSON_PATH).dependencies;
+  const { 'create-svelte': oldCreate } = require('./package.json').dependencies;
+  const newKit = newVersion('@sveltejs/kit');
+  const newVercel = newVersion('@sveltejs/adapter-vercel');
+  const newCreate = newVersion('create-svelte');
+  const branch = `sveltekit-${`${newKit}-${newVercel}-${newCreate}`.replaceAll(
+    '.',
+    '-'
+  )}`;
+
+  if (oldKit === newKit && oldVercel === newVercel && oldCreate === newCreate) {
+    console.log(`SvelteKit package versions did not change; skipping update.`);
+    return;
+  }
```

Comment:
minor nit but we don't use `branch` until we've potentially exited

```suggestion

  if (oldKit === newKit && oldVercel === newVercel && oldCreate === newCreate) {
    console.log(`SvelteKit package versions did not change; skipping update.`);
    return;
  }

  const branch = `sveltekit-${`${newKit}-${newVercel}-${newCreate}`.replaceAll(
    '.',
    '-'
  )}`;
```

Reply:
ty